src/test/kotlin/com/kroffle/knitting/router/DesignRouterTests.kt

```diff
@@ -0,0 +1,80 @@
+package com.kroffle.knitting.router
+
+import com.kroffle.knitting.domain.Design
+import com.kroffle.knitting.domain.DesignRepository
+import com.kroffle.knitting.domain.DesignType
+import com.kroffle.knitting.domain.PatternType
+import com.kroffle.knitting.handler.DesignHandler
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.BDDMockito.given
+import org.springframework.boot.test.autoconfigure.web.reactive.WebFluxTest
+import org.springframework.boot.test.mock.mockito.MockBean
+import org.springframework.http.MediaType
+import org.springframework.test.context.junit4.SpringRunner
+import org.springframework.test.web.reactive.server.WebTestClient
+import reactor.core.publisher.Flux
+import java.time.LocalDateTime
+import java.util.*
+
+@WebFluxTest
+@RunWith(SpringRunner::class)
+class DesignRouterTests {
+    @MockBean
+    lateinit var repo: DesignRepository
+
+    private lateinit var webClient: WebTestClient
+
+    private lateinit var design: Design
+
+    private lateinit var now: LocalDateTime
+
+    @Before
+    fun setUp() {
+        now =  LocalDateTime.now()
+        design = Design(
+            id = UUID.fromString("00000000-0000-0000-0000-000000000000"),
+            name = "test",
+            designType = DesignType.Blanket,
+            patternType = PatternType.Image,
+            stitches = 23.5f,
+            rows = 25.0f,
+            needle = "5.0mm",
+            yarn = null,
+            extra = null,
+            price=0,
+            createdAt = now,
+        )
+
+        val routerFunction = DesignRouter(DesignHandler(repo)).designRouterFunction()
+        webClient = WebTestClient.bindToRouterFunction(routerFunction).build()
+    }
+
+    @Test
+    @Throws(Exception::class)
+    fun `design 리스트가 잘 반환되어야 함`() {
+        given(repo.findAll()).willReturn(Flux.just(design))
+
+        val responseBody: List<Design>? = webClient.get().uri("/designs/")
+                .accept(MediaType.APPLICATION_JSON)
+                .exchange()
+                .expectStatus().isOk
+                .expectBodyList(Design::class.java)
+                .hasSize(1)
+                .returnResult().responseBody
```

Comment:
. 기준으로 줄바꿈 하는 게 좋아보여요! 제안드린 코드에서 webClient는 위로 올려도 좋을 것 같습니다!
```suggestion
        val responseBody: List<Design>? = 
            webClient
                .get()
                .uri("/designs/")
                .accept(MediaType.APPLICATION_JSON)
                .exchange()
                .expectStatus().isOk
                .expectBodyList(Design::class.java)
                .hasSize(1)
                .returnResult().responseBody
```

Reply:
스타일 관련된 코멘트는 Ktlint PR에서 적용하겠습니다!

Reply:
https://github.com/k-roffle/knitting-service/pull/11 의 https://github.com/k-roffle/knitting-service/pull/11/commits/e03e91d9373ca5162fe99e8e650b027ab70f9d61 에서 적용했습니다